Composer/packages/extensions/adaptive-form/src/components/fields/NumberField.tsx

```diff
@@ -1,18 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-import React from 'react';
-import { SpinButton } from 'office-ui-fabric-react/lib/SpinButton';
 import { FieldProps } from '@bfc/extension';
 import formatMessage from 'format-message';
-
+import { SpinButton } from 'office-ui-fabric-react/lib/SpinButton';
+import React from 'react';
 import { FieldLabel } from '../FieldLabel';
 
+const floatNumberOfDecimals = 1;
+
 const getInt = (value: string, step: number) => {
   return parseInt(value, 10) + step;
 };
 
 const getFloat = (value: string, step: number) => {
-  const fixed = (parseFloat(value) + step).toFixed(step !== 0 ? `${step}`.split('.')[1].length : step);
+  const fixed = (parseFloat(value) + step).toFixed(floatNumberOfDecimals);
 
   return parseFloat(fixed);
```

Comment:
Why parse the float to turn it into a float, then back to a string to round it, then back to a float.  How about .round if we are rounding the float?